glossary.md

```diff
@@ -0,0 +1,26 @@
+# Glossary
+An alphabetical list of terms with a short description.
+
+```{glossary}
+
+Actor
+    A component which instantiates one or more Driver classes or does some kind of processing, see {ref}`components.md#actor`.
+
+Component
+    A type of entity, a set of which make up the ECP communication network, see {ref}`components.md#components`.
+
+Coordinator
+    A component primarily concerned with routing/coordinating the message flow, see {ref}`components.md#coordinator`.
+
+Director
+    The main controller orchestrating the whole thing, see {ref}`components.md#director`.
+
+Driver
+    A driver offers a standardized interface to communicate with some hardware (possibly via some intermediate object), see {ref}`components.md#driver`. 
```

Comment:
We're talking quite often about hardware.

What do you think about "device - some piece of hardware controlled via ECP" as a further term?

Reply:
I like it!